deepface/detectors/DetectorWrapper.py

```diff
@@ -149,14 +154,24 @@ def rotate_facial_area(
     Args:
         facial_area (tuple of int): Representing the (x1, y1, x2, y2) of the facial area.
             x2 is equal to x1 + w1, and y2 is equal to y1 + h1
-        angle (float): Angle of rotation in degrees.
-        direction (int): Direction of rotation (-1 for clockwise, 1 for counterclockwise).
+        angle (float): Angle of rotation in degrees. Its sign determines the direction of rotation.
+                       Note that angles > 360 degrees are normalized to the range [0, 360).
         size (tuple of int): Tuple representing the size of the image (width, height).
 
     Returns:
         rotated_coordinates (tuple of int): Representing the new coordinates
             (x1, y1, x2, y2) or (x1, y1, x1+w1, y1+h1) of the rotated facial area.
     """
+
+    # Normalize the witdh of the angle so we don't have to
+    # worry about rotations greater than 360 degrees.
+    # We workaround the quirky behavior of the modulo operator
+    # for negative angle values.
+    direction = (1, -1)[angle < 0]
+    angle = abs(angle) % 360
+    if angle == 0:
```

Comment:
should not we check this as a first task in this function?

Reply:
Actually no cause there's no need to rotate when the result of the mod operation is zero.
For example if the user provides an angle of 720 the test `angle==0` would fail but there's no need to rotate in any case as 720 % 360 == 0